service/service.go

```diff
@@ -2,7 +2,10 @@ package service
 
 import (
 	"context"
+	"github.com/ONSdigital/dp-api-clients-go/zebedee"
+	"github.com/ONSdigital/dp-authorisation/auth"
 	"github.com/ONSdigital/dp-healthcheck/healthcheck"
+	rchttp "github.com/ONSdigital/dp-rchttp"
```

Comment:
`rchttp` is deprecated. Consider `dp-net` instead.

Reply:
Updated to dp-net/http